docs/Installation.md

```diff
@@ -11,7 +11,7 @@ it does it too. You have to confirm everything it will do before it starts.
 ## macOS Requirements
 
 * A 64-bit Intel CPU <sup>[1](#1)</sup>
-* macOS High Sierra (10.13) (or higher) <sup>[2](#2)</sup>
+* macOS Mojave (10.14) (or higher) <sup>[2](#2)</sup>
```

Comment:
See also the contents of the second footnote?

Reply:
Thanks, updated!